nucleus/payara-modules/payara-executor-service/src/main/java/fish/payara/nucleus/executorservice/PayaraExecutorService.java

```diff
@@ -141,37 +136,64 @@ public void event(Event event) {
     }
 
     private void initialiseThreadPools() {
-        int threadPoolExecutorQueueSize = Integer.valueOf(
-                payaraExecutorServiceConfiguration.getThreadPoolExecutorQueueSize());
-        if (threadPoolExecutorQueueSize > 0) {
+        if (isACC()) {
+            
+            // The application client container doesn't use the executor service locally for now.
+            // But create a "mock" executor to be absolutely sure.
             threadPoolExecutor = new ThreadPoolExecutor(
-                    Integer.valueOf(payaraExecutorServiceConfiguration.getThreadPoolExecutorCorePoolSize()),
-                    Integer.valueOf(payaraExecutorServiceConfiguration.getThreadPoolExecutorMaxPoolSize()),
-                    Integer.valueOf(payaraExecutorServiceConfiguration.getThreadPoolExecutorKeepAliveTime()),
-                    TimeUnit.valueOf(payaraExecutorServiceConfiguration.getThreadPoolExecutorKeepAliveTimeUnit()),
-                    new LinkedBlockingQueue<>(threadPoolExecutorQueueSize),
-                    (Runnable r) -> new Thread(r, "payara-executor-service-task"));
+                0,
```

Comment:
Is it possible to move the configured properties to local variables and override them if isAcc passes?